hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/sql/SqlEntityListener.java

```diff
@@ -343,8 +347,25 @@ public void onNftAllowance(NftAllowance nftAllowance) {
     }
 
     @Override
+    @SuppressWarnings({"java:S2259"}) // If nftTransferId is null, this will throw an NPE.  That behavior is correct, for that case.
     public void onNftTransfer(NftTransfer nftTransfer) throws ImporterException {
-        nftTransferState.merge(nftTransfer.getId(), nftTransfer, this::mergeNftTransfer);
+        var nftTransferId = nftTransfer.getId();
+        if (nftTransferId.getSerialNumber() == NftTransferId.WILDCARD_SERIAL_NUMBER) { 
+            // 1. flush nft state to db
+            flushNftState();
+            // 2. nftRepository.updateTreasury
```

Comment:
The comments just duplicate the self descriptive method names and can be removed.
```suggestion
            flushNftState();
```

Reply:
Done.